edx_rbac/migrations/0001_initial.py

```diff
@@ -0,0 +1,73 @@
+# -*- coding: utf-8 -*-
```

Comment:
This migration should be moved into the `test_app` django app.  Since we only need it for testing.  You can see an example of how this was managed in edx-when: https://github.com/edx/edx-when/tree/master/tests/test_models_app/migrations

If you leave it in the edx_rback folder, it will be seen as a migration for this app when we install it in edx-platform and that will cause all sorts of upstream issues.